### PR TITLE
Feature: Added copy underbid keybind + config group

### PR DESCRIPTION
**Pull request by Obsidianninja11**
## Changelog New Features
+ Copy Underbid Keybind. - Obsidian
    * Copies the price of the hovered item in Auction House minus 1 coin into the clipboard for easier under-bidding.

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/inventory/AuctionsHighlighter.kt [hunk 1]
```diff
@@ -63,4 +64,10 @@ class AuctionsHighlighter {
             }
         }
     }
+
+    @SubscribeEvent
+    fun onConfigFix(event: ConfigUpdaterMigrator.ConfigFixEvent) {
+        event.move(24, "inventory.highlightAuctions", "inventory.auctions.highlightAuctions")
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/inventory/AuctionsHighlighter.kt]: You moved three existing config options, you only fixed two in here.
also you need to increase the version number by one

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    @Expose
+    @ConfigOption(name = "Copy Underbid Price",
+        desc = "Copies the price of an item in the \"Create BIN Auction\" minus 1 coin into the clipboard for faster under-bidding.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean copyUnderbidPrice = false;
+
+    @Expose
+    @ConfigOption(name = "Copy Underbid Price",
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java]: Change the feature name. Having two identical names next to each other is confusing.

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        desc = "Copies the price of the hovered item in Auction House minus 1 coin into the clipboard for easier under-bidding.")
+    @ConfigEditorKeybind(defaultKey = Keyboard.KEY_NONE)
+    public int copyUnderbidKeybind = Keyboard.KEY_NONE;
```
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java:46]: the variable name is also very confusing compared to copyUnderbidPrice.
With the config fix event you can also rename the old variable name

**Review by hannibal002 — CHANGES_REQUESTED** [review]
not tested in game

File: src/main/java/at/hannibal2/skyhanni/features/inventory/AuctionsHighlighter.kt (hunk 1 above)
Comment (Obsidianninja11) [reply src/main/java/at/hannibal2/skyhanni/features/inventory/AuctionsHighlighter.kt]: The other one is in AuctionHouseCopyUnderbidPrice.kt

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java (hunk 2 above)
Comment (Obsidianninja11) [reply src/main/java/at/hannibal2/skyhanni/config/features/inventory/AuctionHouseConfig.java]: oops I forgot to change it

**Review by hannibal002 — CHANGES_REQUESTED** [review]
Update CONFIG_VERSION to 25, and use this version in the config fix event

before this pr:
image
with this pr:
image

This migration is still missing:
Copy Underbid Price -> Auto Copy Underbid

**Comment by Obsidianninja11** [comment]
> This migration is still missing:
Copy Underbid Price -> Auto Copy Underbid

It already is migrated though??
image

**Review by hannibal002 — APPROVED** [review]
While testing, I have not bumped both fix events. It's fine now.

**Pull request merged**